packages/features/portfolio/src/components/total-earnings-tooltip/total-earnings-tooltip.tsx

```diff
@@ -20,27 +23,65 @@ interface FirstValueProps {
 }
 
 export const TotalEarningsTooltip = ({
-  toolTipData: { underlyingBaseCurrency, underlying, noteBaseCurrency, note },
+  toolTipData,
 }: TotalEarningsTooltipProps) => {
   const theme = useTheme();
   const { pathname } = useLocation();
+  const {
+    underlyingBaseCurrency,
+    underlying,
+    noteBaseCurrency,
+    note,
+    noteEarnings,
+    nonNoteEarnings,
+  } = toolTipData;
 
   const HoverComponent = () => {
     return (
-      <Box>
-        <ValueContainer sx={{ marginBottom: theme.spacing(1) }}>
-          <FirstValue theme={theme} isNegative={underlying.includes('-')}>
-            {underlying}
-          </FirstValue>
-          <H5>({underlyingBaseCurrency})</H5>
-        </ValueContainer>
-        <ValueContainer>
-          <FirstValue theme={theme} isNegative={note.includes('-')}>
-            {note}
-          </FirstValue>
-          <H5>({noteBaseCurrency})</H5>
-        </ValueContainer>
-      </Box>
+      <div>
+        {noteEarnings && nonNoteEarnings ? (
+          <Box>
+            <FirstValue
+              theme={theme}
+              isNegative={noteEarnings?.includes('-')}
+              sx={{ display: 'flex' }}
+            >
+              <Box sx={{ marginRight: theme.spacing(1) }}>NOTE</Box>
+              <Box>{noteEarnings}</Box>
+            </FirstValue>
+            <ValueContainer
+              sx={{
+                marginTop: theme.spacing(1),
+              }}
+            >
+              <FirstValue theme={theme} sx={{ marginRight: theme.spacing(1) }}>
+                <FormattedMessage defaultMessage={'ORGANIC '} />
+              </FirstValue>
+              <FirstValue
+                theme={theme}
+                isNegative={nonNoteEarnings?.includes('-')}
+              >
+                {nonNoteEarnings}
+              </FirstValue>
+            </ValueContainer>
+          </Box>
+        ) : (
+          <>
```

Comment:
For consistency can we use a Box here as well

Reply:
What is the difference between the first part of the if statement and the second?

Reply:
The first part is only for the total row and shows note usd and combined organic usd values. The second part of the conditional shows four values: note, note usd, token value and the token value usd.